src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelProxyDiscoveryManager.cs

```diff
@@ -248,5 +276,19 @@ private void DiscoverTestsOnConcurrentManager(IProxyDiscoveryManager proxyDiscov
                 EqtTrace.Verbose("ProxyParallelDiscoveryManager: No sources available for discovery.");
             }
         }
+
+        /// <summary>
+        /// Gets all sources which were not discovered(skipped), because of cancelation/abortion of discovery
+        /// </summary>
+        private void getAllNotDiscoveredSources()
```

Comment:
Nit pick: this should be named according to Pascal Case, i.e. `GetAllNotDiscoveredSources`.